docs/conf.py

```diff
@@ -48,7 +48,7 @@ def noop_decorator(*args, **kwargs):
 export_config_help(configs_file_path)
 
 project = "Modin"
-copyright = "2018-2022, Modin"
+copyright = "2018-2022, Modin Developers."
```

Comment:
I think it was supposed to change `Modin Developers` to `Modin` in the NOTICE file as part of our discussion in #3937, but not `Modin` to `Modin Developers` here. cc @devin-petersohn

Reply:
The copyright belongs to Modin devleopers, not Modin, so I think it is correct to be `Modin Developers`. Maybe my understanding is wrong.

Reply:
You're correct.